tests/system/test_file.go

```diff
@@ -0,0 +1,130 @@
+package systest
```

Comment:
Nit: don't put "test" in the filename, `file.go` is better.

Reply:
done